ui/screens/game/controls.lua

```diff
@@ -77,6 +78,9 @@ function buttons.update()
             buttons.layout:mutated()
         end
     end
+    if buttons.hidden then
+        buttons.clear()
+    end
```

Comment:
Doing this in the update is pretty inefficient, I'd rather not add the buttons in the first place when the setting is off.